src/includes/Exports-Include.hs

```diff
@@ -49,6 +56,25 @@
   , setCursorPositionCode
 
     -- * Saving, restoring and reporting cursor position
+    -- | These code sequences are not part of ECMA-48 standard; they are popular,
+    -- but non-portable extensions. E. g., Terminal.app on MacOS
+    -- <https://stackoverflow.com/questions/25879183 does not support them>.
+    -- Somewhat surprisingly, ANSI.SYS does.
+    --
+    -- @terminfo@ package provides a portable way to determine correct sequences
+    -- for cursor positions:
+    --
+    -- >>> import System.Console.Terminfo
+    -- >>> t <- setupTermFromEnv
+    -- >>> getCapability t (tiGetOutput1 "sc") :: Maybe String -- save cursor position
+    -- Just "\ESC7"
+    -- >>> getCapability t (tiGetOutput1 "rc") :: Maybe String -- restore cursor position
+    -- Just "\ESC8"
```

Comment:
I am happy to point people in the direction of other Haskell packages that may be of interest, but I do not think it is for `ansi-terminal` to document the functionality of those packages.

Also, if I understand its description correctly, `terminfo` appears to be of use only on Unix-like operating systems and is not 'portable'.